specification/gedcom.md

```diff
@@ -2831,7 +2831,14 @@ A [registry of component subtags](https://www.iana.org/assignments/language-subt
 
 #### `LANG` (Language) `g7:HEAD-LANG`
 
-The language in which the `Text`-typed payloads of all structures in the document appear, except as superseded by a `g7:LANG`.
+The default language using which to interpert any `Text`-typed payloads of all structures without their own
+`g7:LANG` substructure.  Language information is often used for purposes such as language-specific sorting
+sequences and phonetic, name matching algorithms, spell-checking,
+computer-synthesized speech, Braille transcription, or language translation, and most operating
+systems allow configuring a language via application language settings or "user locales".  Languages
```

Comment:
Because these are **user** locales, shouldn't the first `SUBM`.`LANG` of the `SUBM` record describing the current user be used in lieu of `HEAD`.`LANG` if available? Or am I misunderstanding the meaning of "user" here?